backend/service/service_assets.go

```diff
@@ -383,7 +383,7 @@ func (service *AssetsService) QueryAssetTokenDetail(tokenid string) (vo.AssetTok
 		logger.Error("GetAssetByAddr", logger.String("err", err.Error()))
 		return vo.AssetTokens{}, err
 	}
-	return vo.AssetTokens{
+	ret := vo.AssetTokens{
```

Comment:
add `source` field for vo.AssetTokens, use `source==gateway` condition to judge whether is gateway asset token